app/components/Dashboard/index.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Tab } from 'semantic-ui-react';
 
+import '../../../app/actions/player'
```

Comment:
This won't do what you want it to do. Please look at the containers (in `app/containers` to see how to pass actions down to individual components).